lib/modules/dashboard.js

```diff
@@ -793,8 +799,8 @@ function WidgetObject(widgetOptions) {
 			newSitetable($widgetContent[0]);
 		}
 	};
-	this.error = xhr => {
-		if (xhr.status === 404) {
+	this.error = e => {
+		if (e.message.startsWith('XHR status 404 ')) {
```

Comment:
I don't think so; I think it's just the error object from
```js
throw new Error(`XHR status ${response.status} - url: ${url}`);
```

If there's a nicer approach I'm all ears.